src/simulation/scenario_simulation_inputs.py

```diff
@@ -0,0 +1,396 @@
+
+    vacc_shares_path = BLD / "data" / "vaccinations" / "mean_vacc_share_per_day.yaml"
+    with open(vacc_shares_path) as f:
+        vacc_shares = yaml.safe_load(f)
+
+    vaccination_shares = pd.read_pickle(paths["vaccination_shares"])
+    vaccination_models = _get_vaccination_model_with_new_value_after_date(
+        vaccination_shares,
+        init_start,
+        change_date=SUMMER_SCENARIO_START,
+        new_val=max(vacc_shares.values()),
+        model_name="highest_vacc_share_after_summer_scenario_start",
+    )
+    scenario_inputs = {
+        "vaccination_models": vaccination_models,
+        "contact_policies": _baseline_policies(fixed_inputs),
+        "rapid_test_models": _baseline_rapid_test_models(fixed_inputs),
+        "rapid_test_reaction_models": _baseline_rapid_test_reaction_models(
+            fixed_inputs
+        ),
+    }
+    return scenario_inputs
+
+
+def vaccinations_after_easter_as_on_strongest_week_day(paths, fixed_inputs):
+    """Increase the vaccination rate to that of the best average weekday.
+
+    Averages were taken over the time since familiy physicians started vaccinating.
+    This vaccination rate is extrapolated to every day, including weekends.
+
+    """
+    warnings.warn(
+        "The ambitious vaccination model abstracts from possible delivery constraints."
+    )
+    start_date = fixed_inputs["duration"]["start"]
+    init_start = start_date - pd.Timedelta(31, unit="D")
+
+    vacc_shares_path = BLD / "data" / "vaccinations" / "mean_vacc_share_per_day.yaml"
+    with open(vacc_shares_path) as f:
+        vacc_shares = yaml.safe_load(f)
+
+    vaccination_shares = pd.read_pickle(paths["vaccination_shares"])
+    vaccination_models = _get_vaccination_model_with_new_value_after_date(
+        vaccination_shares,
+        init_start,
+        change_date="2021-04-06",  # Tuesday after Easter Monday
+        new_val=max(vacc_shares.values()),
+        model_name="highest_vacc_share_after_easter",
+    )
+    scenario_inputs = {
+        "vaccination_models": vaccination_models,
+        "contact_policies": _baseline_policies(fixed_inputs),
+        "rapid_test_models": _baseline_rapid_test_models(fixed_inputs),
+        "rapid_test_reaction_models": _baseline_rapid_test_reaction_models(
+            fixed_inputs
+        ),
+    }
+    return scenario_inputs
+
+
+def _get_vaccination_model_with_new_value_after_date(
+    vaccination_shares, init_start, change_date, new_val, model_name
+):
+    vaccination_shares[change_date:] = new_val
+    vaccination_func = partial(
+        find_people_to_vaccinate,
+        vaccination_shares=vaccination_shares,
+        init_start=init_start,
+    )
+    vaccination_models = {model_name: {"model": vaccination_func}}
+    return vaccination_models
+
+
+# ================================================================================
+
+
+def strict_home_office_after_summer_scenario_start(paths, fixed_inputs):
```

Comment:
Do we really need this scenario? Nobody will care about strict home office before next fall. We should keep the helper functions though.

Reply:
I kept the scenario but set the seeds to 0.